src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/services/jobservices/ResetNullEventTimestampMessageJob.java

```diff
@@ -77,18 +76,16 @@ public List<String> run() {
                     eventInformation.getEventTimeStamp()
                 );
 
-                if (eventInformation.getEventTimeStamp() != null) {
-                    messageEntity.setEventTimestamp(eventInformation.getEventTimeStamp());
-                    log.info(
-                        "{} Completed reset main eventTimestamp to {} for message id:{} and case id:{}",
-                        RESET_NULL_EVENT_TIMESTAMP_MESSAGES.name(),
-                        messageEntity.getEventTimestamp(),
-                        messageEntity.getMessageId(),
-                        messageEntity.getCaseId()
-                    );
-                }
+                messageEntity.setEventTimestamp(eventInformation.getEventTimeStamp());
 
-            } catch (JsonProcessingException jsonProcessingException) {
```

Comment:
Any reason why you removed this more specific Exception and introduced a more general one?  Seems like we are going against good practice here.

Reply:
I was testing to see if we could catch other exceptions to increase coverage, but based on what i've tested it only throws jsonprocessingException, so I will change it back